app/views/users/_post.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="post">
+  <div class="post-header">
+    <h3>Post: <%= post.id %></h3>
```

Comment:
- [ ] You should display the post title instead of the post ID.

- [ ] The post on the user show page should be a link to the post instead of just plain text

Reply:
Fixed. Thanks.